Risoluzione del problema di bilanciamento iniziale

cyber-land opened the pull request:
[Issue#1](https://github.com/alessandro-bugatti/batteri/issues/1)
Vengono fatti 100 cicli per ogni ciclo vengono inseriti {numeroBatteri.size()} batteri diversi, in questo modo i batteri sono mescolati e si annullano le differenze perché vengono inseriti simultaneamente

cyber-land committed:
Update mainForm.form

cyber-land committed:
Update mainForm.java

alessandro-bugatti reviewed (commented):
Rimetti il ciclo for come un foreach al posto di un for con indice j

cyber-land committed:
Update mainForm.java

In realtà è necessario un for classico perchè è necessaria la j per definire il colore dei batteri

cyber-land commented:
In realtà è necessario un for classico perchè è necessaria la j per definire il colore dei batteri
